index.html

```diff
@@ -28,30 +28,32 @@
 
         <!-- you can nest rows inside of columns! -->
         <!-- our board has 3 rows, each with 3 columns inside -->
-        <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+        <div class="row" id="row1">
+          <div class="col-xs-4 box col1" id="a"></div>
+          <div class="col-xs-4 box col2" id="b"></div>
+          <div class="col-xs-4 box col3" id="c"></div>
         </div>
 
-        <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+        <div class="row" id="row2">
+          <div class="col-xs-4 box col1" id="d"></div>
+          <div class="col-xs-4 box col2" id="e"></div>
+          <div class="col-xs-4 box col3" id="f"></div>
         </div>
 
-        <div class="row">
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
-          <div class="col-xs-4 box"></div>
+        <div class="row" id="row3">
+          <div class="col-xs-4 box col1" id="g"></div>
+          <div class="col-xs-4 box col2" id="h"></div>
+          <div class="col-xs-4 box col3" id="i"></div>
         </div>
+
+        <br />
+      <div class="row" id="turnMess"></div>
+      <input type="button" value="Reset Game" id="button"/>
       </div>
     </div>
   </div>
   
-  <!-- jquery -->
-
-  <!-- custom script -->
-
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.0.0-alpha1/jquery.min.js"></script>
```

Comment:
Wow, careful you are using the alpha version of jQuery. Most of the examples are for the 1.10 or 1.11 version.

Not a bad thing but something to keep in mind. Some syntax won't work with this version.